src/daft-functions/src/minhash.rs

```diff
@@ -56,12 +74,19 @@ impl ScalarUDF for MinHashFunction {
 }
 
 #[must_use]
-pub fn minhash(input: ExprRef, num_hashes: usize, ngram_size: usize, seed: u32) -> ExprRef {
+pub fn minhash(
+    input: ExprRef,
+    num_hashes: usize,
+    ngram_size: usize,
+    seed: u32,
+    hash_function: HashFunctionKind,
```

Comment:
Would favor this as a `&str` and then implement `HashFunctionKind::from_str` that will convert the user provided string into the hash variant.

Reply:
see https://github.com/Eventual-Inc/Daft/pull/3052#discussion_r1814263514

Reply:
can we talk about an example for how you would like this?